builds/misc/images.yaml

```diff
@@ -151,8 +171,28 @@ jobs:
         name: build
         displayName: Build ($(Build.Configuration))
 
-      - script: scripts/linux/buildDiagnostics.sh $(Build.Configuration)
-        displayName: Build iotedge-diagnostics
+      - template: templates/build-edgelet-linux.yaml
```

Comment:
Do you know why we are using a template yaml for amd64 builds but a bash script for ARM64?

Reply:
Yes. I added build-edgelet-linux template myself to reuse it for iotedged, iotedge-diagnostics and edge-proxy. I will run build script for arm32 and amd64 the similar way it was before to build images for arm32 and amd64. 

arm64 job is different: 
1. not every project for arm32/amd64 is going to be published (e.g. functions)
2. dotnet framework is different (3.0-preview instead of 2.x stable)